### ISSUE TITLE
[DATA] CVE-2023-33951 version information

### ISSUE THREAD

**Issue by DerDakon** [nluedtke/linux_kernel_cves#330]
**Change Type Requested**
Update

**CVE id number**
CVE-2023-33951

**References**
https://github.com/torvalds/linux/commit/a950b989ea29ab3b38ea7f6e3d2540700a3c54e8
https://github.com/torvalds/linux/commit/9ef8d83e8e25d5f1811b3a38eb1484f85f64296c

**Additional context**
The "Fixes" commit sits on a branch that forks from 6.2-rc6 and has been merged into 6.4-rc1 from what I can tell. The commit was cherry-picked into another branch and already merged into 6.2, so there is no affected version >= 6.2. But the version shown for this is still "v5.17-rc1 to v6.4-rc1".

I accept that there are 2 "correct" ways to handle this, and both are confusing.

**Comment by nluedtke** [comment]
Yeah...the data does show that 6.2 is 'fixed' and has the pre-release versions of 6.3 and 6.4 as fixed...but you are right that 6.2 should be the last mainline listed, those other ones are list because the fix code just appears...i agree that i dont like the options here.
